build.gradle

```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.8.0'
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.4.1'
-        def nav_version = '2.4.1'
-        classpath "androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version"
+        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
what is the rationale for adding `org.jetbrains.kotlin:kotlin-gradle-plugin`?

Reply:
Removing the unused Navigation code and its dependency prevented the project from building.
The plugin with id 'kotlin-android' requires this dependency.